ts/screens/messages/MessagesHomeScreen.tsx

```diff
@@ -75,9 +76,9 @@ const styles = StyleSheet.create({
     // Android shadow
     elevation: 5
   },
-  buttonLeft: {
-    padding: 8,
-    marginLeft: -10
+  ioSearch: {
```

Comment:
why we need a style specific for the search button? can we have a more general solution?